core/test/financial-templates/demo/DepositBox.js

```diff
@@ -0,0 +1,303 @@
+const { toWei, utf8ToHex, hexToUtf8, toBN } = web3.utils;
```

Comment:
These tests are great! I like that they are lifecycle to make it clear how to interact with the `DepositBox` and you've commented them well to make the operation clear.